doc/devel/add_new_sink.md

```diff
@@ -18,7 +18,7 @@ We will not merge new code in the Cygnus repository coming from a different path
 
 ### Coding style 
 
-Please, add the `fiware-connectors/flume/telefonica_checkstyle.xml` to you IDE as check style configuration. This XML file contains all the coding style rules accepted by Telefónica.
+Please, add the `fiware-cygnus/telefonica_checkstyle.xml` to you IDE as check style configuration. This XML file contains all the coding style rules accepted by Telefónica.
```

Comment:
I'll perform an exhaustive search when implementing the issue https://github.com/telefonicaid/fiware-connectors/issues/294

NTC